test/unit/auth/user-import-builder.spec.ts

```diff
@@ -230,22 +229,58 @@ describe('UserImportBuilder', () => {
           });
         });
 
-        it('should not throw with valid options and should generate expected request', () => {
+        it('should throw when an invalid hash input order option is provided', () => {
```

Comment:
Add test when hash input order is specified with algorithm that doesn't support it.

Reply:
Sure, to be clear I am putting tests starting at line 383.
Since algorithms in hmacAlgorithms ['HMAC_SHA512', 'HMAC_SHA256', 'HMAC_SHA1', 'HMAC_MD5'] all support specifying hash order.